packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -2,23 +2,33 @@
 
 .personaContainer {
   @include focus-border();
+  border-radius: 15px;
   display: inline-flex;
   align-items: center;
   background: $ms-color-neutralLighter;
   margin: 1px;
   cursor: default;
   user-select: none;
-  max-width: 100%;
+  max-width: 180px;
```

Comment:
@joschect, Not sure what else we can do here. Without a width set here the selected items can grow to be arbitrarily long based on their content. Setting the width here allows ellipsis overflow to happen.